### ISSUE TITLE
Hecke Kernels not displaying correctly for 54.24.a.b

### ISSUE THREAD

**Issue by BarinderBanwait** [LMFDB/lmfdb#4221]
See the Hecke kernels section here: 

https://www.lmfdb.org/ModularForm/GL2/Q/holomorphic/54/24/a/b/

Surely the third line shouldn't be going into the left margin like that?

Also clicking the knowls of those bigints just hangs for at least two minutes.

**Comment by AndrewVSutherland** [comment]
There appears to be a stray quote in the html (in the middle of the first closing </div>)
 
```
<p>This <a title="Newform subspace [cmf.newform_subspace]" knowl="cmf.newform_subspace" kwargs="">newform subspace</a> can be constructed as the kernel of the linear operator \( T_{5}^{3} + 58299525 T_{5}^{2} - \)<a title='[bigint]' knowl='dynamic_show' kwargs='<div style='word-break: break-all'>16823237930372025</div>'>\(16\!\cdots\!25\)</a>\( T_{5} - \)<a title='[bigint]' knowl='dynamic_show' kwargs='<div style='word-break: break-all'>461606347181569682728125</div>'>\(46\!\cdots\!25\)</a>\( \) acting on \(S_{24}^{\mathrm{new}}(\Gamma_0(54))\).
```

**Comment by assaferan** [comment]
It's actually not a stray quote, only too many levels s of quotes. Should be solved in pull request #4225
